libmscore/chordrest.cpp

```diff
@@ -177,14 +177,18 @@ void ChordRest::writeProperties(XmlWriter& xml) const
 
       for (Lyrics* lyrics : _lyrics)
             lyrics->write(xml);
+
+      const int curTick = xml.curTick().ticks();
```

Comment:
This line can be moved down to the line 189.

Reply:
There is an `xml.incCurTick(t);` call between them which will change `xml.curTick()` return value, so this call should be here.